integration/common/openlineage/common/provider/dbt.py

```diff
@@ -172,10 +174,29 @@ def load_catalog(cls, path: str) -> Optional[Dict]:
         except FileNotFoundError:
             return None
 
+    @staticmethod
+    def env_var(var: str, default: Optional[str] = None) -> str:
+        """The env_var() function. Return the environment variable named 'var'.
+        If there is no such environment variable set, return the default.
+
+        If the default is None, raise an exception for an undefined variable.
+        """
+        if var in os.environ:
+            return os.environ[var]
+        elif default is not None:
+            return default
```

Comment:
Shouldn't we throw an exception here? If we don't have that variable, it will fail on dbt execution